data/bun.json

```diff
@@ -11290,42 +11316,42 @@
       "UNZIP": "missing",
       "Unzip": "function",
       "ZLIB_VERNUM": "missing",
-      "Z_BEST_COMPRESSION": "number",
-      "Z_BEST_SPEED": "number",
-      "Z_BLOCK": "number",
-      "Z_BUF_ERROR": "number",
-      "Z_DATA_ERROR": "number",
-      "Z_DEFAULT_CHUNK": "number",
-      "Z_DEFAULT_COMPRESSION": "number",
-      "Z_DEFAULT_LEVEL": "number",
-      "Z_DEFAULT_MEMLEVEL": "number",
-      "Z_DEFAULT_STRATEGY": "number",
-      "Z_DEFAULT_WINDOWBITS": "number",
-      "Z_ERRNO": "number",
-      "Z_FILTERED": "number",
-      "Z_FINISH": "number",
-      "Z_FIXED": "number",
-      "Z_FULL_FLUSH": "number",
-      "Z_HUFFMAN_ONLY": "number",
-      "Z_MAX_CHUNK": "number",
-      "Z_MAX_LEVEL": "number",
-      "Z_MAX_MEMLEVEL": "number",
-      "Z_MAX_WINDOWBITS": "number",
-      "Z_MEM_ERROR": "number",
-      "Z_MIN_CHUNK": "number",
-      "Z_MIN_LEVEL": "number",
-      "Z_MIN_MEMLEVEL": "number",
-      "Z_MIN_WINDOWBITS": "number",
-      "Z_NEED_DICT": "number",
-      "Z_NO_COMPRESSION": "number",
-      "Z_NO_FLUSH": "number",
-      "Z_OK": "number",
-      "Z_PARTIAL_FLUSH": "number",
-      "Z_RLE": "number",
-      "Z_STREAM_END": "number",
-      "Z_STREAM_ERROR": "number",
-      "Z_SYNC_FLUSH": "number",
-      "Z_VERSION_ERROR": "number",
+      "Z_BEST_COMPRESSION": "missing",
```

Comment:
this is unexpected, did bun regress?

can you please manually verify that these symbols are missing? thanks

Reply:
yes.

There in the `constants` (l 11227) but no more available as `require('zlib').x`

![image](https://github.com/user-attachments/assets/d17430df-156a-41a6-9c4c-433b1c7429f6)